client/state/countries/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function getCountries( state, type ) {
+	return state.countries.items[ type ];
+}
+
+export function isFetching( state, type ) {
```

Comment:
I can, but these functions are pretty straightforward. It feels very redundant to write the same thing in a sentence.